### DISCUSSION TITLE
Feature Request: Send individual Grafana Alerts as separate messages

### DISCUSSION THREAD

**Discussion by sdaus** [grafana/grafana#65915] [Alerting]
Currently, Grafana Alerts are grouped together when sent as a notification, making it difficult to quickly identify and address specific issues. It would be great to have an option to send each alert in a separate message.

**Comment by konrad147** [comment]
You can disable or reconfigure grouping in the Root Notification Policy configuration. Set `Group By` to `Disabled` to receive one message per alert
![image](https://user-images.githubusercontent.com/6293563/230557928-b6c8c58b-54cb-4a24-9e8f-2a8696c1d6f4.png)

**Reply by sdaus** [reply]
Thank you! I didn't know I could do this. This is exactly what I was looking for.
